internal/api/handlers/rooms_handler_test.go

```diff
@@ -185,8 +192,19 @@ func TestRoomsHandler_UpdateRoomWithPing(t *testing.T) {
 		require.NoError(t, err)
 
 		for _, invalidStateRawRequest := range invalidStateRawRequests {
-
-			roomStorageMock.EXPECT().GetRoom(gomock.Any(), gomock.Any(), gomock.Any()).Return(gameRoom, nil)
+			// TODO(gabrielcorado): since we're exposing the room manager
```

Comment:
@gabrielcorado I didn't understand this TODO here, could you help me understand?

Reply:
We have to implement a `DoAndReturn`, which for me seems like a "test implementation" for this test. The TODO is not clear what has to be done here, and I'm going to update its description.